### PR TITLE
[14.0][MIG] product attribute set completeness

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
Depends on:

- [ ] https://github.com/OCA/odoo-pim/pull/86

@rousseldenis , @Cedric-Pigeon

### CODE REVIEW

**Comment by rousseldenis** [comment]
@xavier-bouquiaux I think you need to rebase

**Comment by ivantodorovich** [comment]
@xavier-bouquiaux rebase is needed :sweat_smile:

File: test-requirements.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 odoo_test_helper
+
+odoo14-addon-attribute-set-completeness @ git+https://github.com/acsone/odoo-pim.git@14-attribute_set_completeness-xbo#subdirectory=setup/attribute_set_completeness
```
Comment (ivantodorovich) [inline test-requirements.txt:3]: Please mention that it depends on https://github.com/OCA/odoo-pim/pull/86.

Also remember to remove this before merge pls

Comment (simahawk) [reply test-requirements.txt:3]: added to description

**Comment by simahawk** [comment]
@xavier-bouquiaux ping

**Comment by simahawk** [comment]
replaced by https://github.com/OCA/odoo-pim/pull/99

**Pull request closed without merging**